marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mastering/default/matching.sjs

```diff
@@ -20,28 +20,37 @@ const masteringStepLib = require("/data-hub/5/builtins/steps/mastering/default/l
 const requiredOptionProperties = ['matchOptions'];
 const emptySequence = Sequence.from([]);
 
-function filterContentAlreadyProcessed(content, summaryCollection) {
+function filterContentAlreadyProcessed(content, summaryCollection, collectionInfo) {
   const jobID = datahub.flow.globalContext.jobId;
   const filteredContent = [];
   const collectionQuery = cts.collectionQuery(summaryCollection);
   const jobIdQuery = cts.fieldWordQuery('datahubCreatedByJob', jobID);
+  let auditiingNotifiactionsInSourceQuery = false;
```

Comment:
Couple small typos on auditing and notification here, along with notification below on line 31.